modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -77,13 +77,14 @@ If you use Anypoint Connector for File (File Connector) and share part of the pa
 ----
 
 === HTTP and HTTPS
-For HTTP and HTTPS, use the following placeholders, depending on the deployment model:
 
-* `${http.port}` +
-When deploying to CloudHub or Hybrid using HTTP (port=8081) or HTTPS (port=8082)
+For HTTP and HTTPS ports in on-premises deployments, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 
```

Comment:
MuleSoft recommends using > use

Reply:
I think we have to leave "recommends" in this case because they might have a reason to use something else, but we can't go through every scenario for them.